x-pack/plugins/security_solution/public/management/components/console/components/command_list.tsx

```diff
@@ -5,28 +5,64 @@
  * 2.0.
  */
 
-import React, { memo, useMemo } from 'react';
+import React, { memo, useMemo, useCallback } from 'react';
+import styled from 'styled-components';
+import { groupBy, sortBy } from 'lodash';
 import {
   EuiBadge,
+  EuiBasicTable,
+  EuiButtonIcon,
   EuiCode,
   EuiDescriptionList,
   EuiFlexGroup,
+  EuiFlexGrid,
   EuiFlexItem,
   EuiSpacer,
   EuiText,
   EuiTextColor,
+  EuiCallOut,
+  EuiLink,
+  EuiToolTip,
 } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
 import type { CommandDefinition } from '../types';
 import { useTestIdGenerator } from '../../../hooks/use_test_id_generator';
 import { useDataTestSubj } from '../hooks/state_selectors/use_data_test_subj';
+import { useConsoleStateDispatch } from '../hooks/state_selectors/use_console_state_dispatch';
+import { COMMON_ARGS, HELP_GROUPS } from '../service/builtin_commands';
+import { getCommandNameWithArgs } from '../service/utils';
+
+// @ts-expect-error TS2769
+const StyledEuiBasicTable = styled(EuiBasicTable)`
+  margin-top: ${({ theme: { eui } }) => eui.euiSizeS};
+  .euiTableHeaderCell {
+    .euiTableCellContent__text {
+      color: ${({ theme: { eui } }) => eui.euiTextColor};
+      font-size: ${({ theme: { eui } }) => eui.euiFontSize};
+      padding-bottom: ${({ theme: { eui } }) => eui.euiSizeM};
```

Comment:
This looks like a lot of padding on the bottom than on the top. Let's use `sizeS` here.
![Screenshot 2022-07-14 at 11 22 34](https://user-images.githubusercontent.com/1849116/178950348-e792a038-8bb0-43f2-a64f-78c09bcbae4b.png)
![Screenshot 2022-07-14 at 11 22 27](https://user-images.githubusercontent.com/1849116/178950350-c6c8b014-7959-444b-a587-eaec737513e8.png)